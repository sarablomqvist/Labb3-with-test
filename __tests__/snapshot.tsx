/**
 * @jest-environment jsdom
 */
import { render } from "@testing-library/react";
import Home from "@/pages/home/index";
import Show from "@/pages/show/[slug]";

it("Render home unchanged", () => {
  const { container } = render(<Home shows={[]} />);
  expect(container).toMatchSnapshot();
});

it("Render show unchanged", () => {
  const { container } = render(<Show show={{ name: "A", summary: "B" }} />);
  expect(container).toMatchSnapshot();
});
