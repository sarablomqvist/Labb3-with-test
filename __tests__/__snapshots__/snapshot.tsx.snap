// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render home unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <main>
      <h1
        class="title"
      >
        Welcome to top TV Shows with G!
      </h1>
      <p
        class="description"
      >
        Click a movie to learn more!
      </p>
      <div
        class="grid"
      />
    </main>
    <footer
      class="footer"
    >
      <a
        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        Powered by
         
        <span
          class="logo"
        >
          <img
            alt="Vercel Logo"
            data-nimg="1"
            decoding="async"
            height="16"
            loading="lazy"
            src="/vercel.svg"
            style="color: transparent;"
            width="72"
          />
        </span>
      </a>
    </footer>
  </div>
</div>
`;

exports[`Render show unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      A
    </h1>
    <div>
      B
    </div>
    <a
      href="/home"
    >
      <button>
        Back
      </button>
    </a>
    <h2>
      0
    </h2>
    <button
      type="button"
    >
      +
    </button>
  </div>
</div>
`;
